### PR TITLE
Pia 2918 camera screen

### PR DESCRIPTION
**Pull request by chriskryn**
- added new UI for iPhones and iPads
- removed tooltips
- refactored camera screen

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            "ginicapture.camera.flashButtonLabel",
+            comment: "Flash button"),
+            giniconfiguration: giniConfiguration)
+        flashButton.iconView.image = UIImageNamedPreferred(named: "flashOn")
+    }
+
+    /**
+     Disable all camera buttons except capture button.
+     */
+    func configureCameraButtonsForQRCodeTip() {
```
Comment (a-szotyori) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift:39]: [Clarification] Is this still needed?

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        captureButton.isEnabled = true
+        flashButton.isEnabled = true
+        flashButton.isSelected = giniConfiguration.flashOnByDefault
+        fileUploadButton.actionButton.isEnabled = false
+        fileUploadButton.actionLabel.isEnabled = false
+        fileUploadButton.actionButton.isUserInteractionEnabled = false
+    }
+
+    func configureCameraWhenTooltipDismissed() {
```
Comment (a-szotyori) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift:48]: [Clarification] Is this still needed?

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController+ToolTip.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
+//
+//  Camera2ViewController+Extension.swift
```
Comment (a-szotyori) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController+ToolTip.swift:2]: [Suggestion] File is named `...+ToolTip`, but it is used to show the QR code detected popup. As we won't have tooltips anymore I recommend to use `...+Popups` instead. Oh and also change the file name in the comment here 😉.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,321 @@
+//
+//  Camera2ViewController.swift
```
Comment (a-szotyori) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift:2]: [Clarification] Will this end up being renamed to `CameraViewController` to replace the old one?

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift (hunk 1 above)
Comment (chriskryn) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift:39]: You are right, It's not needed. I will remove it

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift (hunk 4 above)
Comment (chriskryn) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift:2]: Not sure, I left the old version as agreed during discussion. We can always replace/rename it.

Comment (a-szotyori) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift:2]: I think I missed that discussion. I will ask for a reminder in our daily today 🙈.

**Review by zladzeyka — COMMENTED** [review]
Great job @chriskryn 👏🏻😊🎉
I've found 2 more litle things:
Screenshot 2022-09-20 at 15 24 39
Could you please check this warning in iPad.xib?
And the label in the frame is missing 😉
Screenshot 2022-09-20 at 15 30 16

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPreviewView.swift [hunk 5]
```diff
@@ -21,55 +21,60 @@ final class CameraPreviewView: UIView {
...
-    
+
     func drawGuides(withColor color: UIColor) {
         guideColor = color
         createGuides()
         createGrayFrame()
     }
-    
+
     fileprivate func createGuides() {
```
[the remark this answers is not in the document]
Comment (chriskryn) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPreviewView.swift:63]: @zladzeyka The guides are used only in old camera screen version. Should I remove it?

**Comment by chriskryn** [comment]
changes in https://github.com/gini/gini-mobile-ios/pull/152

**Pull request closed without merging**